packages/frontend/src/components/notes/note/newNoteModal.tsx

```diff
@@ -118,7 +118,6 @@ const NewNoteModal: React.FC<NewNoteModalProps> = ({
         <Text size={'1.25rem'} margin="1.5%">
           Type
         </Text>
-        {/* This is just as a start. Will use Tailwind for this and not some ugly select tag haha */}
```

Comment:
This comment is funny, it really reminds me of that reading we did on working with legacy code. Temporary fixes are never temporary.